tests/Fixer/LanguageConstruct/FunctionToConstantFixerTest.php

```diff
@@ -126,6 +126,30 @@ public function provideTestCases()
                 '<?php echo phpversion(); echo php_sapi_name(); echo M_PI;',
                 ['functions' => ['pi', 'phpversion']],
             ],
+            'get_class => T_CLASS' => [
```

Comment:
Please add a test case with a trait instead of a class and one with a call to `get_class()` from outside a class. I think the code should not be changed for the latter as the result is not the same (`false` + warning vs. `""`).